packages/eslint-plugin-jest/src/rules/valid-expect.js

```diff
@@ -16,7 +16,16 @@
 
 import type {EslintContext, CallExpression} from './types';
 
+const matchers = require('jest-matchers/build/matchers');
```

Comment:
Not sure if it's OK to reach into here? Are there any better ways to ensure we match against all possible built-in matchers?